app/views/portal/show.rb

```diff
@@ -420,6 +421,30 @@ def labels
 
     private
 
+    def document_hierarchy
```

Comment:
Assignment Branch Condition size for document_hierarchy is too high. [20/15]<br>Method has too many lines. [11/10]